### PR TITLE
Offers page: keep alive until canceled or completed

### PR DESCRIPTION
**Pull request by dparikh**

### CODE REVIEW

File: src/runtime/offers-flow.js [hunk 1]
```diff
@@ -91,6 +89,8 @@ export class OffersFlow {
       if (skuId) {
         return new PayStartFlow(this.deps_, skuId).start();
       }
+    }).catch(() => {
+      this.dialogManager_.completeView(this.activityIframeView_);
```
Comment (dvoytenko) [inline src/runtime/offers-flow.js]: Can't swallow the error. Let's rethrow.

File: src/runtime/offers-flow.js [hunk 2]
```diff
@@ -77,7 +76,6 @@ export class OffersFlow {
   start() {
     // If result is due to OfferSelection, redirect to payments.
     this.activityIframeView_.acceptResult().then(result => {
-      this.dialogManager_.completeView(this.activityIframeView_);
```
Comment (dvoytenko) [inline src/runtime/offers-flow.js]: Keep it until switch to messaging.

File: src/runtime/offers-flow.js (hunk 1 above)
Comment (dparikh) [reply src/runtime/offers-flow.js]: Done

File: src/runtime/offers-flow.js (hunk 2 above)
Comment (dparikh) [reply src/runtime/offers-flow.js]: Done

**Pull request merged**